### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ddavydenko**

### CODE REVIEW

**Comment by ddavydenko** [comment]
@vdantu , @vrakesh , @frankfliu , could you please review this? There is no code here, just a template for office hours sign up.

File: .github/ISSUE_TEMPLATE/office-hours-template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (vdantu) [inline .github/ISSUE_TEMPLATE/office-hours-template.md]: Just curious, how would this file be used?

**Comment by ddavydenko** [comment]
I intend this file to serve as template for creating new issues when
customers will be signing up for office hours. I am doing this with
ModelServer first as I can get this template committed into master and try
to use it before submitting corresponding proposal with the similar
template for mxnet repo.

File: .github/ISSUE_TEMPLATE/office-hours-template.md (hunk 1 above)
Comment (frankfliu) [reply .github/ISSUE_TEMPLATE/office-hours-template.md]: How this template can be picked up?
Github seems only pick ISSUE_TEMPLATE.md file.
Can we enable Wiki on Github and allow people sign-up there? Wiki is also a git, we should be able to look at the wiki update history if there is abuse of it.

**Pull request closed without merging**